pkg/aws/policies.go

```diff
@@ -674,14 +676,75 @@ func parsePolicyDocument(path string) (PolicyDocument, error) {
 	return doc, nil
 }
 
+func isOCMRole(roleName *string) bool {
+	return strings.Contains(aws.StringValue(roleName), OCMRoleInfix)
```

Comment:
Can we use the same one from polciies.go or change it to some common place. we should be using the same constant while creating/listing/deleting so if it changes. SO i think it is safer to check the tag as well

Reply:
Changed to use the same variable from **policies.go** (used in creating).
I've tried to check the tags as well, but it's really slowing down the command (~2 min), we have ~500 roles ending up in
500 requests to the **AWS client** to get the tags for each role.
![image](https://user-images.githubusercontent.com/57869309/152676668-486d7534-62ab-4d28-91d8-2db726eeb4c2.png)
As for checking the role name, [once created it can't be changed](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user.html), so if we are consistent on our side it should work.
WDYT?